explcheck/src/explcheck-common.lua

```diff
@@ -0,0 +1,17 @@
+-- Transform a filename into a state object that can be passed to the processing steps.
```

Comment:
The number of files is growing. Perhaps move the comments inside functions and place a single comment on top of each Lua file, explaining its purpose?

Reply:
Fixed in commit fbc0d00 from this PR.